docs/index.md

```diff
@@ -12,7 +12,7 @@
  
 ## Getting Started
 
-Start with [**Project Setup**](Tutorials/Getting Started.md) and keep clicking next, this will move through the tutorials in a sensible order.
+Start with [**Project Setup**](/Basics/Getting Started.md) and keep clicking next, this will move through the tutorials in a sensible order.
```

Comment:
This isn't really true any more. This will move you through all of the network integrations in order which isn't something anyone wants!